server.go

```diff
@@ -254,6 +256,38 @@ func (rh *moduleHandler) handlePanic(wr http.ResponseWriter, req *http.Request,
 			return
 		}
 	} else {
+		writeErrorToResponse(err, debug.Stack(), wr)
 		log.Printf("[UncaughtPanic] %s\n-- ERROR --\nUncaught panic in route module %T: %+v\n-- STACK TRACE --\n%s", req.URL, rh.module, err, debug.Stack())
 	}
 }
+
+func writeErrorToResponse(errorString error, stackTrace []byte, wr http.ResponseWriter) {
+	tmpl := `
+    <html>
+        <head>
+            <title>Error Occured</title>
+        </head>
+        <body>
+            <style>
+                <h1>Error</h1>
+                <p>{{.Error}}</p>
+                <pre>{{.StackTrace}}</pre>
+            </style>
```

Comment:
Can we remove the style tag ??

Reply:
Hey, yeah. I think because the content is nested in the style tag it won't get rendered in the browser. Something like this:

```html
<style>
</style>

<pre>{{.StackTrace}}</pre>
```

I'm on holiday this week so I may be slow to respond. But let me know if you have other questions!